README.md

```diff
@@ -133,7 +116,97 @@ include the `restPort` configuration to tell the exporter which port to use.
 * [Chart-based sample](samples/kubernetes/end2end): This is an end-to-end sample that shows you the steps to set up monitoring for WebLogic domains using Prometheus and Grafana. Prometheus and Grafana are installed with Helm charts.
 
 
+# Web Application
```

Comment:
Web Application -> Web application (our standard in the WKO and related docs is to use sentence capitalization in titles)
You need to add text under this heading. -> (From the Oracle Style Guide) When writing headings, remember the following guidelines: Do not follow one heading with another without having intervening text. Provide an explanation of the content that follows.